### ISSUE TITLE
Question about the dataset split

### ISSUE THREAD

**Issue by HUANGLIZI** [nikhilroxtomar/TGANet#2]
Can you provide detailed division of training set, test set and verification set? Thanks so much!
